docs/uccignore.md

```diff
@@ -5,4 +5,23 @@ add-on recursively overrides the output folder.
 
 It is expected to be placed in the same folder as `globalConfig` file to have effect.
 
-You will see a warning message in case ignored file is not found in the output folder.
+Uccignore supports wildcard expressions, thanks to which we can find all files matching a specific pattern.
+
+e.g. for given file structure 
+
+```
+...
+└── lib
+    └── 3rdparty
+        ├── linux
+        │   ├── pycache.pyc
+        ├── linux_with_deps
+        │   ├── pycache.pyc
+        └── windows
+            ├── pycache.pyc
+```  
+
+we can remove all `.pyc` files by adding `lib/**/pycache.pyc` to .uccignore file. 
+If we want to remove only for one specific platform we need to provide the exact path e.g. **`lib/3rdparty/windows/pycache.pyc`**.
+
+In case .uccignore file is not found you will see a warning message.
```

Comment:
Let's rephrase it to that in case no file found according to the pattern, then you will see a warning message.